src/JavascriptTrait.php

```
@@ -0,0 +1,344 @@
+ *   Given I visit "/legacy-page"
+ *   # Errors collected but not asserted - test passes
+ * @endcode
+ */
+trait JavascriptTrait {
+
+  /**
+   * Registry of JavaScript errors collected during scenario execution.
+   *
+   * @var array<string, array<int, array<string, mixed>>>
+   */
+  protected array $javascriptErrorRegistry = [];
```

PHPMD (2.15.0)

Avoid excessively long variable names like $javascriptErrorRegistry. Keep variable name length under 20. (undefined)

(LongVariable)